src/main/java/org/synyx/urlaubsverwaltung/sicknote/statistics/SickNoteStatisticsService.java

```diff
@@ -79,8 +86,9 @@ private Function<Map.Entry<Person, List<SickNote>>, SickNoteDetailedStatistics>
         return personListEntry ->
         {
             final Person person = personListEntry.getKey();
-            final String personnelNumber = Optional.of(basedataForPersons.get(new PersonId(person.getId())).getPersonnelNumber()).orElse("");
-            final List<String> departments = Optional.of(departmentsForPersons.get(new PersonId(person.getId()))).orElse(List.of());
+            final PersonId personId = new PersonId(person.getId());
+            final String personnelNumber = basedataForPersons.getOrDefault(personId, new PersonBasedata(personId, "", "")).getPersonnelNumber();
+            final List<String> departments = departmentsForPersons.getOrDefault(personId, List.of());
```

Comment:
Hat List.of() einen Vorteil gegenüber Collections.emptyList()? Die letztere ist immutable